docs/Alerting.md

```diff
@@ -0,0 +1,133 @@
+# Alerting
```

Comment:
Some of this looks like it might be more broadly applicable to the Tools team, or other specific Tools (e.g. Composer), rather than just to the Fronts tool. Is there any value in storing the more generic information in a more central place? We have a [runbook repo](https://github.com/guardian/editorial-tools-support).

Reply:
To be entirely honest, it wasn't written for this PR.  It was written when I was annoyed that it wasn't written anywhere.  Happy to move it to anywhere more appropriate.